stdlib/public/Platform/CMakeLists.txt

```diff
@@ -68,11 +63,28 @@ foreach(sdk ${SWIFT_SDKS})
               "-DGLIBC_INCLUDE_PATH=${GLIBC_INCLUDE_PATH}"
               "-DGLIBC_ARCH_INCLUDE_PATH=${GLIBC_ARCH_INCLUDE_PATH}")
 
+      list(APPEND glibc_modulemap_target_list ${glibc_modulemap_target})
+
+      # If compiling for a non-native SDK, generate a native modulemap without a sysroot prefix
+      # The prefixed one is still the 'glibc.modulemap' which we'll look at during the build process
+      if(NOT "${SWIFT_SDK_${sdk}_PATH}" STREQUAL "/")
```

Comment:
@karwa I think so.
Or just hardcode to exclude `"ANDROID"` as @gribozavr said would be OK for now.